docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,807 @@
+
+```
+I: 254 [DL]Device Configuration:
+I: 257 [DL] Serial Number: TEST_SN
+I: 260 [DL] Vendor Id: 65521 (0xFFF1)
+I: 263 [DL] Product Id: 32768 (0x8000)
+I: 267 [DL] Hardware Version: 1
+I: 270 [DL] Setup Pin Code: 20202021
+I: 273 [DL] Setup Discriminator: 3840 (0xF00)
+I: 278 [DL] Manufacturing Date: (not set)
+I: 281 [DL] Device Type: 65535 (0xFFFF)
+```
+
+In above printout, the _discriminator_ is `3840 (0xF00)` and the _setup PIN
+code_ is equal to `20202021`.
+
+### Step 6: Commission Matter device into existing IP network
+
+Before communicating with the Matter device, first have it join the existing IP
+network. Matter devices may use different commissioning channel. Typically,
+devices which are not yet connected to the target IP network use Bluetooth LE as
+the commissioning channel. On the other hand, if the device is already joined to
+the IP network it is enough to only commission it to the Matter network over IP
+protocol.
+
+#### Commissioning over Bluetooth LE
+
+This section describes how your device can join the existing IP network over
+Bluetooth LE and then be commissioned into a Matter network.
+
+After connecting the device over Bluetooth LE, the controller will print the
+following log:
+
+```
+Secure Session to Device Established
+```
+
+This log will mean that the PASE (Password-Authenticated Session Establishment)
+session using SPAKE2+ protocol has been established.
+
+##### Commissioning into Thread network over Bluetooth LE
+
+To commission the device to the existing Thread network, use the following
+command pattern:
+
+```
+$ ./chip-tool pairing ble-thread <node_id> hex:<operational_dataset> <pin_code> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<operational_dataset\>_ is the Operational Dataset determined in the
+    step 3.
+-   _<pin_code\>_ and _<discriminator\>_ are device-specific keys determined in
+    the step 4.
+
+##### Commissioning into Wi-Fi network over Bluetooth LE
+
+To commission the device to the existing Wi-Fi network, use the following
+command pattern:
+
+```
+$ ./chip-tool pairing ble-wifi <node_id> <ssid> <password> <pin_code> <discriminator>
+```
+
+In this command:
+
+-   _<node_id\>_ is the user-defined ID of the node being commissioned.
+-   _<ssid\>_ and _<password\>_ are credentials determined in the step 3.
+-   _<pin_code\>_ and _<discriminator\>_ are device-specific keys determined in
+    the step 4.
+
+If you prefer the hexadecimal format, use the `hex:` prefix. For example:
+
+```
+$ ./chip-tool pairing ble-wifi <node_id> hex:<ssid> hex:<password> <pin_code> <discriminator>
+```
+
+> **Note:** The _<node_id>_ can be provided as a hexadecimal value with the `0x`
+> prefix.
+
+#### Commissioning into a network over IP
+
+This option is available when the Matter device is already present in an IP
+network, but it has not been commissioned to a Matter network yet.
```

Comment:
```suggestion
This option is available when the Matter device has already joined an IP
network, but it has not been commissioned to a Matter network yet.

To commission the device, you can use either the setup PIN code or the discriminator, both of which you obtained in the step 5. Alternatively, you can also use a QR code payload.
```

Reply:
It's rather PIN code **or** PIN code **and** discriminator.

Reply:
OK, can you make edits based on this?

Reply:
Done already